utils/query.js

```diff
@@ -0,0 +1,26 @@
+const bcd = require('./bcd');
+
+/**
+ * Get a subtree of compat data.
+ *
+ * @param {string} path Dotted path to a given feature (e.g., `css.properties.background`)
+ * @param {*} [data=bcd] A tree to query. All of BCD, by default.
+ * @returns {*} A BCD subtree
+ * @throws {ReferenceError} For invalid identifiers
+ */
+function query(path, data = bcd) {
+  const pathElements = path.split('.');
+  let lookup = data;
+  while (pathElements.length) {
+    const next = pathElements.shift();
+    lookup = lookup[next];
+    if (lookup === undefined) {
+      throw ReferenceError(
```

Comment:
`new` keyword not required?

Reply:
Apparently it works without it, but it does look off, doesn't it? Fixed with 8e094cf4.